docs/contributing/contributing-code/contributing-code-prerequisites/README.md

```diff
@@ -26,100 +59,138 @@ This is the list of core dependencies to install for the most common tasks. In g
 - [Python](https://www.python.org/downloads/)
 - [Golangci-lint](https://golangci-lint.run/usage/install/#local-installation)
 - [jq](https://jqlang.github.io/jq/download/)  
-- Make  
-  
-  **Linux**: Install the `build-essential` package:
-  ```bash
-  sudo apt-get install build-essential
-  ```
-  **Mac**:
-  Using Xcode:
-  ```bash  
-  xcode-select --install
-  ```
-  Using Homebrew:
-  ```bash  
-  brew install make
-  ```
+- Make
```

Comment:
Is there a reason to remove these instructions? We found mac users get stuck on this step.

Reply:
In this section I kept the list of tools as an overview. The deatiled steps for installing `make` are described in the local insatllation section starting in lines 151. 
I thought it makes sense not to bother the contributors that any way use GitHub Codespaces or dev containers with these steps as this might be confusing.